src/picongpu/include/plugins/adios/restart/LoadParticleAttributesFromADIOS.hpp

```diff
@@ -82,8 +83,9 @@ struct LoadParticleAttributesFromADIOS
 
         for (uint32_t n = 0; n < components; ++n)
         {
+            OpenPMDName<T_Identifier> openPMDName;
             std::stringstream datasetName;
-            datasetName << particlePath << T_Identifier::getName();
+            datasetName << particlePath << openPMDName();
```

Comment:
because it is not yet refactored to use the optional openPMD particle patches (but our old particle info object).